community/contributing/nightly-builds/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Nightly Builds
+layout: single
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+Bugs affecting more than one service commonly surface only when the whole system is running. Our continuous integration system builds and runs a suite of integration tests against Spinnaker nightly on real cloud provider infrastructure to detect these bugs.
+
+> **Access the CI system here**: [https://builds.spinnaker.io](https://builds.spinnaker.io)
+>
+> Requires membership in the `build-cops` GitHub Team to access the system.
```

Comment:
What requires membership? Just accessing builds.spinnaker.io?

Reply:
Changed to "Viewers must be a member of the `build-cops` GitHub Team."